qa/os/src/test/java/org/elasticsearch/packaging/util/Distribution.java

```diff
@@ -57,12 +64,17 @@ public boolean isPackage() {
         return packaging == Packaging.RPM || packaging == Packaging.DEB;
     }
 
+    public boolean isDocker() {
```

Comment:
Why is this needed? It doesn't seem any clearer than `packaging == Packaging.DOCKER`

Reply:
To be honest I was copying the pattern of the existing code. e.g. `isArchive()` is only called in one place 🤷‍♂. I can remove it.